### PR TITLE
Change backend from Django to Node.js

### PR DESCRIPTION
**Pull request by ayush-goyal**
Changes
----
- Switches from Django to Nodejs
- Sets up database with Prisma
- Use Apollo Server

### CODE REVIEW

File: server/src/util/slack.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { RequisitionStatus } from "@prisma/client";
+import { WebClient } from "@slack/web-api";
+import { GraphQLError } from "graphql";
+import { projectReferenceString, requisitionReferenceString, statusToString } from "../api/resolvers/common";
+import { prisma } from "../common";
+
+const web = new WebClient(process.env.SLACK_API_TOKEN);
+
+export const sendSlackNotification = async (requisitionId: number) => {
```
Comment (evan10s) [inline server/src/util/slack.ts:9]: You don't have to fix this in this PR, but in the future it'd be better to create a more generic way to send notifications with any content to any user

File: server/src/util/slack.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            approvals: {
+                orderBy: {
+                    id: "desc"
+                }
+            }
+        }
+    });
+
+    if (!process.env.SLACK_API_TOKEN || !process.env.ROOT_URL || !requisition?.createdBy.slackId || requisition.status == RequisitionStatus.DRAFT) {
+        return;
```
Comment (evan10s) [inline server/src/util/slack.ts:26]: You should log a warning when this case is hit

**Review by evan10s — APPROVED** [review]
looks good!  going forward, you should consider ways to split up files that are likely to get large, like the query/mutation resolvers, as more complex functionality is added in the future.  Similarly, for stuff like Slack notifications, it would be better to create generic notification interfaces that can be implemented so that the content of a notification isn't tied to the code that actually sends the notification.

File: server/src/util/slack.ts (hunk 1 above)
Comment (ayush-goyal) [reply server/src/util/slack.ts:9]: Yeah will do

**Pull request merged**